src/components/AutoSuggestion/AutoSuggestion.tsx

```diff
@@ -1,16 +1,18 @@
 import React, { useEffect, useMemo, useState } from "react";
 import { Classes as BlueprintClassNames } from "@blueprintjs/core";
-import CodeMirror, { Editor as CodeMirrorEditor, Position } from "codemirror";
+import { Rect } from "@uiw/react-codemirror";
```

Comment:
Why don't we use our `CodeEditor` editor as foundation for `CodeAutocompleteField` as specialized version?